packages/oui-inline-adder/src/inline-adder.controller.js

```diff
@@ -0,0 +1,29 @@
+import { hasAttributeValue } from "@oui-angular/common/component-utils";
+export default class {
+    constructor ($element, $q) {
+        "ngInject";
+        this.$element = $element;
+        this.$q = $q;
+    }
+
+    $onInit () {
+        if (!hasAttributeValue(this.$element[0], "on-add")) {
+            throw new Error("ovh-ui-angular: The required callback 'on-add' is not provided.");
```

Comment:
This could be thrown automatically with a `throwErrorifMissingAttribute(element, attributeName)` function for example

Reply:
It could even be injected I think